ftplugin/latex-suite/texviewer.vim

```diff
@@ -733,6 +743,10 @@ if has('python') && Tex_GetVarValue('Tex_UsePython')
 	python import sys, re
 	exec "python sys.path += [r'". s:path . "']"
 	python import outline
+elseif has('python3') && Tex_GetVarValue('Tex_UsePython')
+	python3 import sys, re
+	exec "python3 sys.path += [r'". s:path . "']"
+	python3 import outline
```

Comment:
Yeah, I was just trying to write my thesis and meet a deadline, I seem to recall I got a bit hacky by the end of that day.